lib/write.ml

```diff
@@ -121,7 +121,7 @@ let float_needs_period s =
 
   The _fast version is faster but often produces unnecessarily long numbers.
 *)
-let write_float_fast ob x =
+let write_float_fast ob x = (* unused *)
```

Comment:
Can't really say that it is unused, since this is in the `.mli`, so it might be used by some code on OPAM.

Though it might very well be that nobody is using it, in which case we should consider removing it.

Reply:
At least according to [sherlocode](https://sherlocode.com/?q=write_float_fast), nothing on opam uses this. Since it's hidden from the documentation as well, I suppose there doesn't have to be any stability guarantees for this part of the interface. Maybe something at some point needed this (also atdgen?).

If you'd like, I can remove this comment from this PR, but the actual removal of the function is best to be left as a separate matter.

Reply:
Thanks for the research. I agree removing this would not make sense in the scope of this PR, but if you happen to have a moment after this is merged to remove the unused functions that would be nice. Otherwise I'll probably get to it… eventually.

Of course, there might be some non-public code that uses these functions, but accumulating tons of functions without a clear usecase is somewhat of an anti-pattern and makes maintenance harder.